web_src/js/utils/html.test.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @eslint-community/eslint-comments/disable-enable-pair */
+/* eslint-disable github/unescaped-html-literal */
+import {html, htmlEscape, htmlRaw} from './html.ts';
+
+test('html', async () => {
+  expect(html`<a>${'<>&\'"'}</a>`).toBe(`<a>&lt;&gt;&amp;&#39;&quot;</a>`);
+  expect(html`<a>${htmlRaw('<img>')}</a>`).toBe(`<a><img></a>`);
+  expect(html`<a>${htmlRaw`<img ${'&'}>`}</a>`).toBe(`<a><img &amp;></a>`);
+  expect(htmlEscape(`<a></a>`)).toBe(`&lt;a&gt;&lt;/a&gt;`);
+});
```

Comment:
Just put the `/* eslint-enable github/unescaped-html-literal */` on the last line of this file.

Reply:
No idea how to correctly address your concern, feel free to edit this PR directly.

Reply:
Or maybe this commit https://github.com/go-gitea/gitea/pull/34911/commits/6c949b018a3d76c3234f1934a54f0b646aabf3c8 works as your expectation?